test/lib/code_corps/model/task_test.exs

```diff
@@ -46,6 +64,26 @@ defmodule CodeCorps.TaskTest do
       {:ok, %Task{created_at: created_at, modified_at: modified_at}} = Repo.insert(changeset)
       assert created_at == modified_at
     end
+
+    test "sets the order when the task is not archived and position is set" do
+      project = insert(:project)
+      task_list = insert(:task_list)
+      insert(:task, task_list: task_list, order: 1)
+      user = insert(:user)
+      changes = Map.merge(@valid_attrs, %{
+        position: 1,
+        project_id: project.id,
+        task_list_id: task_list.id,
+        user_id: user.id
+      })
+      changeset = Task.create_changeset(%Task{}, changes)
+      assert changeset.valid?
+      {:ok, %Task{order: order}} = Repo.insert(changeset)
+
+      # We really want to test the order is set, but we have no good way to
+      # test this since the column default is `0`
+      assert order !== 0
```

Comment:
Ah so assert order fails.  Would have thought since refute only handles nil and false that assert would have worked!  :walking_man:

Reply:
`assert order` succeeds but that's insufficient to tell us anything. Since the default is `0` it could just be set to `0`, but instead it will receive a random ranked value from `EctoOrdered`.